src/saving/FileHelpers.cs

```diff
@@ -43,4 +44,22 @@ public static bool Exists(string path)
         using var directory = new Directory();
         return directory.FileExists(path);
     }
+
+    /// <summary>
+    ///   Try to create a file with write access
+    /// </summary>
+    /// <param name="path">Path to file</param>
+    /// <returns>File status</returns>
```

Comment:
I think a better description would be like:
```suggestion
    /// <summary>
    ///   Tests if it is possible to create/write a file on the given path
    /// </summary>
    /// <param name="path">Path to file</param>
    /// <returns>File write access status, <see cref="Error.Ok"/> if successful</returns>
```